.github/workflows/nightly-rosetta-pax-build.yaml

```diff
@@ -105,6 +100,28 @@ jobs:
       BASE_IMAGE: ${{ needs.metadata.outputs.BASE_IMAGE_ARM64 }}
     secrets: inherit
 
+  publish-build-badge:
+    needs: [metadata, amd64, arm64]
+    uses: ./.github/workflows/_publish_badge.yaml
```

Comment:
Note that `_publish_badge.yaml` is deprecated. In the future, we will generate Badge JSONs as part of the job sitrep artifact, and then collectively publish them using `_fianlize.yaml`.

Reply:
I agree we should move to sitrep, but given how large this PR is, I'd like to tackle that in follow-up PR. This shouldn't change the status-quo; in fact it should fix things b/c this badge is referenced on the front-page readme and it was somehow removed in a PR so it hasn't been getting updated

Reply:
Sounds good. I just wanted to raise the issue so that you are aware of the coming change.